tests/e2e/index.spec.ts

```diff
@@ -61,9 +64,20 @@ test.describe("Index page", () => {
     });
 
     test("tooltips are shown", async ({ page }) => {
-        const firstRegion = await getFirstRegion(page);
+        const firstRegion = await getNthRegion(page, 1);
         await firstRegion.hover();
         await expect(await page.innerText(".leaflet-tooltip-pane")).toContain("Bengo");
         await expect(await page.innerText(".leaflet-tooltip-pane")).toContain("FOI: 0.0134");
     });
+
+    test("selecting country fades colours of other countries", async ({ page }) => {
+        const firstRegion = await getNthRegion(page, 1);
+        const colour = await firstRegion.getAttribute("fill");
+        const stroke = await firstRegion.getAttribute("stroke");
+        const secondRegion = await getNthRegion(page, 1000);
```

Comment:
Doesn't this get the 1000th region, not the second?

Reply:
yes my bad! i wanted to do second region and them i realised idk how many admin1 features the first region even has, have gone for last region now, think that is safer!

Reply:
I thought you meant 'second' as in 'the second one we're looking at here'! :laughing: